JProxies/doc/index.md

```diff
@@ -0,0 +1,44 @@
+## JProxy
+JProxy lets you use Java-like syntax to access fields and methods in Java. You can:
+* Get field values
+* Set field values
+* Call methods
+* Create static proxies to access static members
+Primitive types and strings are converted to Julia objects on field accesses and method returns and converted back to Java types when sent as arguments to Java methods.
+*NOTE: Because of this, if you need to call Java methods on a string that you got from Java, you'll have to use `JProxy(str)` to convert the Julia string to a proxied Java string*
+To invoke static methods, set static to true (see below).
```

Comment:
I wrote this before I added the `@class` macro -- I think it should change to this:

`To invoke static methods, use @class (see below).`